server/src/main/java/dev/slimevr/tracking/processor/skeleton/HumanSkeleton.java

```diff
@@ -1380,33 +1380,34 @@ public boolean isTrackingRightArmFromController() {
 		return rightHandTracker != null && rightHandTracker.getHasPosition() && !forceArmsFromHMD;
 	}
 
-	protected Tracker[] getTrackersToReset() {
-		return new Tracker[] {
-			trackerPreUpdate(this.neckTracker),
-			trackerPreUpdate(this.chestTracker),
-			trackerPreUpdate(this.waistTracker),
-			trackerPreUpdate(this.hipTracker),
-			trackerPreUpdate(this.leftUpperLegTracker),
-			trackerPreUpdate(this.leftLowerLegTracker),
-			trackerPreUpdate(this.leftFootTracker),
-			trackerPreUpdate(this.rightUpperLegTracker),
-			trackerPreUpdate(this.rightLowerLegTracker),
-			trackerPreUpdate(this.rightFootTracker),
-			trackerPreUpdate(this.leftLowerArmTracker),
-			trackerPreUpdate(this.rightLowerArmTracker),
-			trackerPreUpdate(this.leftUpperArmTracker),
-			trackerPreUpdate(this.rightUpperArmTracker),
-			trackerPreUpdate(this.leftHandTracker),
-			trackerPreUpdate(this.rightHandTracker),
-			trackerPreUpdate(this.leftShoulderTracker),
-			trackerPreUpdate(this.rightShoulderTracker),
-		};
+	public List<Tracker> getLocalTrackers() {
+		return List
+			.of(
+				trackerPreUpdate(this.neckTracker),
+				trackerPreUpdate(this.chestTracker),
+				trackerPreUpdate(this.waistTracker),
+				trackerPreUpdate(this.hipTracker),
+				trackerPreUpdate(this.leftUpperLegTracker),
+				trackerPreUpdate(this.leftLowerLegTracker),
+				trackerPreUpdate(this.leftFootTracker),
+				trackerPreUpdate(this.rightUpperLegTracker),
+				trackerPreUpdate(this.rightLowerLegTracker),
+				trackerPreUpdate(this.rightFootTracker),
+				trackerPreUpdate(this.leftLowerArmTracker),
+				trackerPreUpdate(this.rightLowerArmTracker),
+				trackerPreUpdate(this.leftUpperArmTracker),
+				trackerPreUpdate(this.rightUpperArmTracker),
+				trackerPreUpdate(this.leftHandTracker),
+				trackerPreUpdate(this.rightHandTracker),
+				trackerPreUpdate(this.leftShoulderTracker),
+				trackerPreUpdate(this.rightShoulderTracker)
```

Comment:
Worth noting, `trackerPreUpdate` is totally useless now and can be completely removed. Doesn't need to be done in this PR, but could be convenient since it's already being modified.